### PR TITLE
Add store(), create(), type, discovery internals

### PR DESCRIPTION
**Pull request by marcoscaceres**
Part of #65 

The following tasks have been completed:

- [x] [Modified Web platform tests](https://github.com/web-platform-tests/wpt/pull/46642)

Implementation commitment:

- [X] WebKit - Already implemented
- [X] Chromium  - Already implemented. 
- [ ] Gecko (link to issue)

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/digital-credentials/pull/121.html" title="Last updated on Jun 13, 2024, 1:15 AM UTC (2bbf497)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/digital-credentials/121/deff75c...2bbf497.html" title="Last updated on Jun 13, 2024, 1:15 AM UTC (2bbf497)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -269,6 +272,45 @@ <h3>
       The <dfn data-dfn-for="DigitalCredential">data</dfn> member is the
       credential's response data.
     </p>
+    <h3>
+      [[\Store]](credential, sameOriginWithAncestors) internal method
+    </h3>
+    <p>
+      When invoke, the <dfn class="export" data-dfn-for=
```
Comment (TallTed) [inline index.html:279]:
```suggestion
      When invoked, the <dfn class="export" data-dfn-for=
```

File: index.html [hunk 2]
```diff
@@ -269,6 +272,45 @@ <h3>
...
+      "DigitalCredential">[[\Store]](credential, sameOriginWithAncestors)</dfn>
+      MUST call the default implementation of {{Credential}}'s
+      {{Credential/[[Store]](credential, sameOriginWithAncestors)}} internal
+      method with the same arguments.
+    </p>
+    <h3>
+      [[\Create]](origin, options, sameOriginWithAncestors) internal method
+    </h3>
+    <p>
+      When invoke, the <dfn class="export" data-dfn-for=
```
Comment (TallTed) [inline index.html:289]:
```suggestion
      When invoked, the <dfn class="export" data-dfn-for=
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by marcoscaceres** [comment]
This matches WebKit and Chromium so far... Tests are at https://github.com/web-platform-tests/wpt/pull/46642

File: index.html [hunk 3]
```diff
@@ -269,6 +272,45 @@ <h3>
...
+      [[\Create]](origin, options, sameOriginWithAncestors) internal method
+    </h3>
+    <p>
+      When invoked, the <dfn class="export" data-dfn-for=
+      "DigitalCredential">[[\Create]](origin, options,
+      sameOriginWithAncestors)</dfn> internal method MUST run the following
+      steps:
+    </p>
+    <ol class="algorithm">
+      <li>Return `null`.
```
Comment (samuelgoto) [inline index.html:295]: Can't we also call the default implementation of `Create` here, as you did to `Store`?

The default implementation seems to return `null` too?

https://www.w3.org/TR/credential-management-1/#algorithm-create-cred

Comment (marcoscaceres) [reply index.html:295]: Yes, you are right. We can call directly into the default one.

**Review by samuelgoto — APPROVED** [review]
LGTM

**Pull request merged**